lib/price-import.coffee

```diff
@@ -134,6 +137,15 @@ class PriceImport extends ProductImport
     debug 'About to send %s requests', _.size(posts)
     Promise.settle(posts)
 
+  ###
+   * returns true if product does not have any staged changes
+   * and is published
+  ###
+  _canBePublished: (product) ->
+    if !product.hasStagedChanges and product.published
```

Comment:
Manual approval is not always required. In my opinion it is just covering single requirement and it is not common use case. It should be more flexible and work with options flag called: ```publishing``` or ```publishStrategy``` then value could be ```always```, ```publishedOnly```, ```stagedAndPublishedOnly```